### PR TITLE
[ONNX] GRU and RNN operators.

### PR DESCRIPTION
**Pull request by arogowie-intel**

### CODE REVIEW

File: ngraph/src/ngraph/frontend/onnx_import/utils/recurrent.cpp [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+{
+    namespace onnx_import
+    {
+        namespace recurrent
+        {
+            OpInputMap::OpInputMap(const onnx_import::Node& node, std::size_t gates_count)
+            {
+                const auto& ng_inputs = node.get_ng_inputs();
+
+                m_map[OpInput::X] = ng_inputs.at(0);
```
[the remark this answers is not in the document]
Comment (arogowie-intel) [reply ngraph/src/ngraph/frontend/onnx_import/utils/recurrent.cpp:39]: Good point. Maybe not all are required, I'll review it once again and try to assert only what's necessary.

**Comment by itikhono** [comment]
LGTM, but OpenVINO don't support W/R/B as inputs, please add several tests where W/R/B are Constants.

**Comment by arogowie-intel** [comment]
> 
> 
> LGTM, but OpenVINO don't support W/R/B as inputs, please add several tests where W/R/B are Constants.

@itikhono I've just added one such test case, and although I don't get errors about transformation,  IE_CPU produces bad results.

**Comment by itikhono** [comment]
> > LGTM, but OpenVINO don't support W/R/B as inputs, please add several tests where W/R/B are Constants.
> 
> @itikhono I've just added one such test case, and although I don't get errors about transformation, IE_CPU produces bad results.

:( We need to find the root cause. I'll investigate it today.

**Comment by arogowie-intel** [comment]
The IE_CPU test with W,R as constants will fail due to the lack of support for multioutput in nG IE backend. Nevertheless the first output provides correct results.

**Pull request merged**